CHANGELOG.md

```diff
@@ -1,3 +1,12 @@
+## v2.0.0-rc.1 / 2017-10-17
```

Comment:
Without the `v` in the Changelog (don't ask me, was a long discussion ...).

Reply:
Oh, the previous RC had a `v`, removed it now.